piet-gpu/shader/binning.comp

```diff
@@ -15,11 +15,6 @@ layout(set = 0, binding = 0) buffer AnnotatedBuf {
     uint[] annotated;
 };
 
-// This is for scanning forward for right_edge data.
```

Comment:
It should be possible to remove the descriptor, as we no longer need to give the binning kernel access to the state buffer. Since this is a cleanup, interested in doing that as well?

Reply:
This should be done now!